pkgs/development/libraries/matrix-sdk-crypto-nodejs/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, napi-rs-cli, nodejs, libiconv }:
+
+stdenv.mkDerivation rec {
+  pname = "matrix-sdk-crypto-nodejs";
+  version = "0.1.0-beta.1";
+
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "matrix-rust-sdk";
+    rev = "${pname}-v${version}";
+    hash = "sha256-t9iaIfiw8m86Z0AvXgDaNVuy8odGSOf3gU/UtR6bevk=";
+  };
+
+  patches = [ ./cargo-lock.patch ];
```

Comment:
This can be removed next update, as a `Cargo.lock` has been added since this release.